src/public/snippets/head.ejs

```diff
@@ -1,6 +1,15 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <meta name="author" content="Victoria">
+<meta property="og:locale" content="en_GB">
+<meta property="og:type" content="website">
+<meta property="og:title" content="WaifuVault Temporary File Hosting">
+<meta property="og:url" content="https://waifuvault.moe/">
+<meta property="og:site_name" content="WaifuVault Temporary File Hosting">
+<meta property="og:description" content="File Hosting for nerds">
```

Comment:
better description i think

something like "quick, free and easy temporary file hosting"

Reply:
also for twitter handle

Reply:
Updated